src/crypto/CHIPCryptoPAL.h

```diff
@@ -30,6 +30,8 @@
 #include <core/CHIPError.h>
 #include <support/CodeUtils.h>
 
+#include "hsm/CHIPCryptoPALHsm_config.h"
```

Comment:
Same as above. Consider protecting header include with #if CHIP_CRYPTO_HSM == 1

Reply:
Sure This Inclusion is also protected